components/global/product_switcher_tip/product_switcher_tip.tsx

```diff
@@ -114,7 +113,7 @@ export class ProductSwitcherTip extends React.PureComponent<Props, State> {
         });
     }
 
-    render() {
+    render(): JSX.Element | null {
```

Comment:
This is automatic. It doesn't help anyone to put this here.

Reply:
if that's the case why do we have the before mentioned eslint rule in place? It is constantly throwing warnings about functions/methods not having a return type. It is done very inconsistently in the codebase, so this should probably be put to discussion as well.